dqr_agent.py

```diff
@@ -0,0 +1,191 @@
+from rom_wrapper import ROMWrapper
+from nes_py.wrappers import JoypadSpace
+from nes_py._image_viewer import ImageViewer
+from nes_py.app.play_human import play_human
+from nes_py.app.cli import _get_args
+import time
+
+import numpy as np
+from itertools import count
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torch.distributions import Categorical
+
+args = _get_args()
+
+# required arg
+rom_path = args.rom
+
+
+# create the environment
+env = ROMWrapper(rom_path)
+
+SHOULD_RENDER = True
+FLAT_STATE_SIZE = 2048
+GAMMA = 0.2
+log_interval = 100
+num_steps_per_episode = 2250
+reward_threshold = 1500.0
+# DQR STUFF
```

Comment:
@boshea93 this is an example agent based on the PyTorch example for reinforcement learning